### DISCUSSION TITLE
Bug in the `Create a new repository` UI.

### DISCUSSION THREAD

**Discussion by danBamikiya** [community/community#4532] [Other Feature Feedback, Questions, & Ideas] [answered]
The hovercard that shows if a new repository name is available is too transparent to show the '{repo name} is available' message:

![image](https://user-images.githubusercontent.com/58262528/124766821-2a3c8b80-df2f-11eb-8b8e-402ec7ac364c.png)

I dug into the developers tool a little, I suggest giving the `form .js-with-permission-fields` div a `margin-top: 32px;` styling when the input field is clicked like so:

![github_feedback_issue_2](https://user-images.githubusercontent.com/58262528/124771386-febba000-df32-11eb-8b08-84cde0acedfc.png)

And a sleek transition of `transition: 0.2s ease-in;` could be added to make the effect smoother. :tada:

@jonrohan @mdo Think this is possible?

Edit:
@bdresser Think this is possible?

**Comment by davidstaheli** [comment, marked as answer]
@danBamikiya, thank you for reporting this.  It should be fixed now.

![image](https://user-images.githubusercontent.com/1767415/131114086-baebf50c-b475-48ea-a7c0-24e1c2c3b6c5.png)

**Reply by danBamikiya** [reply]
@davidstaheli Thanks for fixing this!
